source/wp-content/themes/wporg-developer/scss/main.scss

```
@@ -594,6 +598,9 @@
 				display: block;
 				margin: 0 auto;
 				opacity: 0.4;
+				display: flex;
+    			align-items: center;
+    			flex-direction: column;
```

```suggestion
				display: flex;
				align-items: center;
				flex-direction: column;
```
Looks like some spaces slipped in there at the start.

---

Can't tell the difference in IDE without linting... Thanks for catching it!
Fixed.